src/Network/MPD/Commands/CurrentPlaylist.hs

```diff
@@ -63,6 +65,10 @@ clear = A.runCommand A.clear
 delete :: MonadMPD m => Position -> m ()
 delete = A.runCommand . A.delete
 
+-- | Remove a range of songs from the current playlist.
```

Comment:
@iyefrat Can you add haddock since annotation of 0.9.3 ? Also, can you update the changelog ?

Reply:
At the time i thought that since these functions were documented as added a few versions ago it would be pointless to say that they were added again, but rethinking this makes it obvious that it makes it all the more necessary to document this as to avoid confusion. Wouldn't it make more sense to say since 0.9.2 as that version has not been released yet?

Reply:
@psibi it has occurred to me that you might not be notified automatically by my response, so I'm doing it manually for good measure.

Reply:
@iyefrat 

> but rethinking this makes it obvious that it makes it all the more necessary to document this as to avoid confusion

Yeah, I don find it very useful.

> Wouldn't it make more sense to say since 0.9.2 as that version has not been released yet?

Yes, that's right. Sorry for my wrong information before. I assumed 0.9.2 has been released. (I guessing adding v0.9.2 with unreleased heading in changelog would make it better!)